lib/travis/build/addons.rb

```diff
@@ -56,6 +56,8 @@ def addon(name, config)
           const = self.class.const_get(name.to_s.camelize)
           const.new(script, sh, data, config) if const && run_addon?(const)
         rescue NameError
+        rescue
+          raise Travis::Build::AddonConfigError.new(name)
```

Comment:
Does this need to be a bare `rescue`?  Would `rescue StandardError => e` with `e` _at least_ being logged be acceptable?